src/daft-scan/src/lib.rs

```diff
@@ -126,6 +114,62 @@ impl ScanTaskBatch {
         }
     }
 
+    pub fn concat(scan_tasks: &[Self]) -> ScanTask {
```

Comment:
This function should not be used until we want to start batching with the scheduler

Reply:
Removed the function